CHANGELOG.md

```diff
@@ -88,7 +99,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - First marketplace release
 
-[Unreleased]: https://github.com/owncloud/files_primary_s3/compare/v1.3.0...master
+[Unreleased]: https://github.com/owncloud/files_primary_s3/compare/v1.4.0...master
+[1.4.0]: https://github.com/owncloud/files_primary_s3/compare/v1.3.1...v1.4.0
+[1.3.1]: https://github.com/owncloud/files_primary_s3/compare/v1.3.0...v1.3.1
```

Comment:
https://github.com/owncloud/files_primary_s3/compare/v1.3.0...v1.3.1
"We couldn’t figure out how to compare these references, do they point to valid commits?"

There hasn't been a 1.3.1

Reply:
Fixed in PR #646